RELEASING.md

```diff
@@ -113,13 +201,13 @@ export PREVIOUS_SDK_VERSION=v0.<MINOR>.<PATCH>
 gh release create sdk/python/${SDK_VERSION} --generate-notes --notes-start-tag sdk/python/${PREVIOUS_SDK_VERSION} --draft
 ```
 
-#### 2/4. Clean up release notes
+#### 2/5. Clean up release notes
 
-- Add link to PyPI, e.g. 🐍 https://pypi.org/project/dagger-io/0.1.1/
-- If there is a blog post (see **4/4.**) add a link to it, e.g.
-  📝 https://dagger.io/blog/python-sdk
-- If there is a video (see **4/4.**) add a link to it, e.g.
-  🎬 https://www.youtube.com/watch?v=c0bLWmi2B-4
+- Add link to PyPI, e.g. `🐍 https://pypi.org/project/dagger-io/0.1.1/`
```

Comment:
The existing python release also has a docs link like `📖 https://dagger-io.readthedocs.io/en/sdk-python-v0.2.0/`: https://github.com/dagger/dagger/releases/tag/sdk%2Fpython%2Fv0.2.0

Reply:
Resolved as a commit amend & force push. The link is now in the Python SDK release notes list.